build.sbt

```diff
@@ -149,6 +87,51 @@ lazy val tests = project
   .dependsOn(dom)
   .enablePlugins(ScalaJSPlugin, BuildInfoPlugin, NoPublishPlugin)
 
+def mkJSEnv(options: Capabilities) = {
+  val config = SeleniumJSEnv
+    .Config()
+    .withMaterializeInServer("target/selenium", s"http://localhost:$fileServicePort/target/selenium/")
+  new SeleniumJSEnv(options, config)
+}
+
+lazy val chromeTests = tests.settings(Test / jsEnv := mkJSEnv(new ChromeOptions().setHeadless(true)))
+lazy val firefoxTests = tests.settings(Test / jsEnv := mkJSEnv(new FirefoxOptions().setHeadless(true)))
```

Comment:
The main idea is to have a separate test project for each browser. This avoids the complexity of changing sbt settings to cycle through the test environments.

Reply:
Hmm, I'm not as good at sbt as I thought I was 😅